### DISCUSSION TITLE
how to reset all values to 0

### DISCUSSION THREAD

**Discussion by armdan** [emporia-vue-local/esphome#148] [General]
can you please help me reset the values?
I want clean start of readings

thanks

**Comment by clowrey** [comment]
That involves editing the underlying database of Home Assistant directly unfortunately... I wish they made it easy - someday - but not today. I recommend just renaming them in ESPHome if you need to start from 0.

**Comment by armdan** [comment]
I did clean install of Hass but I still get old values. I guess I will rename them.
thanks
